fiat-rust/Cargo.toml

```
@@ -1,8 +1,13 @@
 [package]
 name = "fiat-rust"
 version = "0.1.0"
-authors = ["Zoe Paraskevopoulou <zoopar@fb.com>"]
+authors = ["Fiat library authors <jgross@mit.edu>"]
 edition = "2018"
```

Should this be updated?
```suggestion
 edition = "2020"
```

---

No, this is really [2018](https://doc.rust-lang.org/edition-guide/editions/index.html).